code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -328,9 +328,10 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 /datum/clockwork_scripture/guvax/scripture_effects()
 	for(var/mob/living/L in hearers(1, get_turf(invoker))) //Affects silicons
-		if(!is_servant_of_ratvar(L))
+		if(!is_servant_of_ratvar(L) && is_eligible_servant(L))
```

Comment:
Uh, add_servant_of_ratvar() alreacy checks if they're eligible and makes a fluff message if they are.